lib/v2/qipamaijia/index.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const rootUrl = 'https://www.qipamaijia.com';
+
+module.exports = async (ctx) => {
+    const cate = ctx.params.cate ?? '/';
+    const url = `${rootUrl}/${cate}`;
```

Comment:
`url` will become `https://www.qipamaijia.com//`
```suggestion
    const cate = ctx.params.cate ?? '';
    const url = `${rootUrl}/${cate}`;
```

Reply:
Thx